tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -808,7 +855,8 @@ def _execute_sts_job(sts_client, target_project, source_bucket_name,
                 'bucketName': sink_bucket_name
             },
             "transferOptions": {
-                "deleteObjectsFromSourceAfterTransfer": True,
+                #"deleteObjectsFromSourceAfterTransfer": False,
+                "deleteObjectsFromSourceAfterTransfer": False if is_retain_source_bucket.lower() == 'true' else True      
```

Comment:
replace it as "deleteObjectsFromSourceAfterTransfer": False if is_retain_source_bucket else True since is_retain_source_bucket is bool type